### PR TITLE
french bibstrings

### PR DESCRIPTION
**Pull request by maieul**

### CODE REVIEW

File: archaeologie-french.lbx [hunk 1]
```diff
@@ -24,14 +24,32 @@
...
-%\DeclareBibliographyStrings{%
-%  inherit          = {english},
-%    translationof = {{Translation\addspace of}{Transl\adddot\addspace of}},
-%  reference = {{Article}{Art\adddot}},
-%  reviewof = {{critique\addspace du}{critique\addspace du}},
-%  origtitle = {{original title}{orig\adddot title}},
-%  firstprint = {{first\addspace edition}{first\addspace edition}},
-%}
+\DeclareBibliographyStrings{%
+  inherit          = {english},
```
Comment (LukasCBossert) [inline archaeologie-french.lbx]: is `english` done purposely ? not `inherit = {french}` ?

Comment (maieul) [inline archaeologie-french.lbx]: you are right. should be corrected to french. I let you do it ;-)

**Comment by LukasCBossert** [comment]
I had to do some corrections, but now it works properly. thanks again.

```
\ProvidesFile{archaeologie-french.lbx}%
               [2016/03/17 v1.42  archaeologie --%
                biblatex fuer Archaeologen, Historiker und Philologen, lbx-Datei]
\InheritBibliographyExtras{french}
\DefineBibliographyExtras{french}{\restorecommand\mkbibnamefamily}

\NewBibliographyString{origtitle}
\NewBibliographyString{firstprint}
\NewBibliographyString{reviewof}
\NewBibliographyString{reference}
\NewBibliographyString{translationof}
\NewBibliographyString{talk}


\DeclareBibliographyStrings{%
  inherit          = {french},
    translationof = {{traduction\addspace de}{trad\adddot\addspace de}},
  reference = {{article}{art\adddot}},
  reviewof = {{recension\addspace de}{rec\adddot\addspace de}},
  origtitle = {{titre original}{titre orig.\adddot}},
  firstprint = {{première\addspace édition}{première\addspace éd\adddot}},
  talk = {{conférence}{conférence}},
  fromamerican     ={{depuis\addspace l'anglais américain} {depuis\addspace l'anglais américain}},
  frombrazilian    = {{depuis\addspace le\addspace brésilien}{depuis\addspace le\addspace brésilien}},
  fromcroatian     = {{depuis\addspace le\addspace croatien}{depuis\addspace le\addspace croatien}},
  fromdanish       = {{depuis\addspace le\addspace danois}{depuis\addspace le\addspace danois}},
  fromdutch        = {{depuis\addspace le\addspace néerlandais}{depuis\addspace le\addspace néerlandais}},
  fromenglish      = {{depuis\addspace l'anglais}{depuis l'anglais}},
  fromfinnish      = {{frin Finnish}{depuis\addspace le\addspace Finnish}},
  fromfrench       = {{depuis\addspace le\addspace français}{depuis\addspace le\addspace français}},
  fromgerman       ={{depuis\addspace l'allemand} {depuis\addspace l'allemand}},
  fromgreek        = {{depuis\addspace le\addspace grec}{depuis\addspace le\addspace grec}},
  fromitalian      = {{depuis\addspace l'italien}{depuis l'italien}},
  fromlatin        = {{depuis\addspace le\addspace latin}{depuis\addspace le\addspace latin}},
  fromnorwegian    = {{depuis\addspace le\addspace norvégien}{depuis\addspace le\addspace norvégien}},
  fromportuguese   = {{depuis\addspace le\addspace portugais}{depuis\addspace le\addspace portugais}},
  fromrussian      = {{depuis\addspace le\addspace russe}{depuis\addspace le\addspace russe}},
  fromspanish      = {{depuis\addspace l'espagnole}{depuis\addspace l'espagnol}},
  fromswedish      = {{depuis\addspace le\addspace suédois}{depuis\addspace le\addspace suédois}},
}

```

**Comment by maieul** [comment]
talk = {{conf�rence}{conf�rence}},

> talk = {{conf�rence}{conf.}},

**Comment by LukasCBossert** [comment]
`talk 		    = {{conférence}{conf.}},`
right?!

**Comment by maieul** [comment]
right

**Pull request merged**